src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java

```diff
@@ -0,0 +1,42 @@
+package io.github.vhoyon.vramework.utilities;
+
+import java.util.HashMap;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public final class TimerManager {
+
+	private static HashMap<String,Timer> timers;
+
+	private TimerManager(){}
+
+	/**
+	 * @param timerName Name of the timer
+	 * @param delay Delay (in milliseconds) before doing the action
+	 * @param action Action to do after the delay was finished, without being called again, which resets the delay
+	 */
+	public static void manage(final String timerName, final int delay, TimerTask action){

```

Comment:
Can we change the method name to `schedule`, therefore matching the way `java.util.Timer` already does it? It makes more sense anyway.